### ISSUE TITLE
Training/Validation loss diagram

### ISSUE THREAD

**Issue by andy-96** [sel118/LaneAF#11]
Thanks for the great work!

I am trying to retrain LaneAF on CUlane, however, it is converging really slowly (at least for the first 8 epochs) and without waiting for 6 days to run 60 epochs, I wanted to ask, whether it is possible to share your diagrams on training/validation loss.

Thanks in advance!

**Comment by arangesh** [comment]
Here are the plots from a finished training run on CULane:

![train_loss](https://user-images.githubusercontent.com/13139124/116793305-e171dd80-aa7a-11eb-9afa-4868d48a4a86.jpg)
![val_loss](https://user-images.githubusercontent.com/13139124/116793306-e2a30a80-aa7a-11eb-8d7d-2a535bc18844.jpg)
![trainval_acc_f1](https://user-images.githubusercontent.com/13139124/116793307-e33ba100-aa7a-11eb-9ba1-363426a0a101.jpg)
